docs/log-correlation.asciidoc

```diff
@@ -33,16 +33,20 @@ See the relevant https://www.elastic.co/guide/en/apm/agent/index.html[Agent docu
 [float]
 ==== Add APM identifiers to your logs
 
-Once log correlation is enabled,
-you must ensure your logs contain APM identifiers.
+Once log correlation is enabled, you must ensure your logs contain APM identifiers.
```

Comment:
This edit is not necessary. APM doc contributors follow the convention of adding line breaks at the end of each phrase. It's a standard convention for ASCIIDOC source though not all teams follow it.